lib/variations.js

```diff
@@ -8,6 +9,11 @@ const variations = {
   'default'() {
     Rec.start();
   },
+  'autoOpenPanel'() {
+    simplePrefs.prefs.autoOpenPanel = true;
+    Rec.start();
+  },
+  'contol'() {},
```

Comment:
this is a typo :)  I might suggest 'observe-only'.

Rename 'default' to "passive-panel" maybe?  These three names are doing to show up in the all the analysis and should self document